packages/PriceLockup/PriceLockup.modules.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@tds/core-colours/colours'; // be sure to add core-colours as a dependency first
+
+.hairlineWrapper {
+  padding: 0.5rem;
+}
+
+.priceValueSign {
+  display: flex;
+  composes: h1 from '../Heading/Heading.modules.scss';
+  h1, h2 {
+    color: $color-text;
+  }
+}
+
+.priceWrapper {
+  display: flex;
+  align-items: flex-end;
+}
+
+.rateWrapperSmall {
```

Comment:
Please remove these three classes that use `padding-bottom`, since core-box can handle spacing for us.

Reply:
removed